test/el-test.js

```diff
@@ -126,3 +127,129 @@ describe('yo polymer:el --path test', function () {
     );
   });
 });
+
+describe('yo polymer:el TDD test', function () {
+
+  before(function (done) {
+    helpers.run(path.join(__dirname, '../el'))
+      .inDir(path.join(__dirname, './tmp'))
+      .on('ready', function (generator) {
+        var js = 'WCT.loadSuites([\'my-greeting-basic.html\', \'my-list-basic.html\']);';
+        fs.outputFileSync(path.join(__dirname, './tmp/app/test/index.html'), js);
+        assert.file(path.join(__dirname, './tmp/app/test/index.html'));
+      })
+      .withArguments(['x-tag'])
+      .withPrompt({
+        includeImport: false,
+        testType: 'TDD'
+      })
+      .on('end', done);
+  });
+
+  it('creates expected files', function () {
+    var expected = [
+      'app/test/index.html',
+      'app/test/x-tag-basic.html',
+      'app/elements/x-tag/x-tag.html'
+    ];
+
+    assert.file(expected);
+  });
+
+  it('Modifies index.html properly (with previous items)', function () {
+    assert.fileContent(
+      'app/test/index.html',
+      /'my-list-basic.html', 'x-tag-basic.html'\]\);/ // Was inserted at the end of the array properly
+    );
+  });
+  
+  it('does not include any words for BDD', function() {
+    assert.noFileContent(
+      'app/test/x-tag-basic.html',
+      /(describe\()|(before\()|(it\()/  // These words only appear in BDD test. This is a TDD
+    )
+  });
+});
+
+describe('yo polymer:el BDD test', function () {
+
+  before(function (done) {
+    helpers.run(path.join(__dirname, '../el'))
+      .inDir(path.join(__dirname, './tmp'))
+      .on('ready', function (generator) {
+        var js = 'WCT.loadSuites([]);';
```

Comment:
You can nest the before blocks like this: https://gist.github.com/robdodson/6e1672f6dc7a3f00b678 That should ensure that the app generator is finished before the el generator runs. Then you should be able to test with the index file.